model/BeneficiaryTransferDestinationResponse.ts

```diff
@@ -0,0 +1,12 @@
+export interface BeneficiaryTransferDestinationResponse {
+	beneficiaryId: string
+	name: string
+}
+
+export namespace BeneficiaryTransferDestinationResponse {
+	export function is(
+		value: BeneficiaryTransferDestinationResponse | any
+	): value is BeneficiaryTransferDestinationResponse {
+		return typeof value == "object" && typeof value.beneficiaryId == "string" && typeof value.name == "string"
+	}
+}
```

Comment:
this one's wrong, it extends the AccountDetails one

Reply:
tried to emulate the extend bit in is function